test/Fencer/Types/Test.hs

```diff
@@ -99,7 +99,7 @@ test_parseJSONDomainAtLeastOneDescriptor :: TestTree
 test_parseJSONDomainAtLeastOneDescriptor =
   testCase "DomainDefinition has to have at least one descriptor" $
     assertEqual "parsing DomainDefinition failed"
-      (Left "Error in $.descriptors: Expected a NonEmpty but got an empty list")
```

Comment:
This change is probably due to aeson.

Reply:
If you're referring to the fact that the list of descriptors has to be non-empty, it's due to the data type for domain definitions and its field `domainDefinitionDescriptors`: https://github.com/juspay/fencer/blob/ff939faf43e0c82c54896ade320c7f25e9bb3427/lib/Fencer/Types.hs#L141

Reply:
Oh, no, I'm referring to the fact that the wording of the error message has changed.